Doc/glossary.rst

```diff
@@ -191,7 +193,7 @@ Glossary
       A variable defined in a class and intended to be modified only at
       class level (i.e., not in an instance of the class).
 
-      Class variables can be specified as such through
+      Variables can be specified as expected to be class variables through
       :term:`type hints <type hint>`.
```

Comment:
If you're going to be this specific I recommend calling out `ClassVar` by name.

Reply:
I agree that this was getting too specific. I left the class variable definition, though, because I think it's useful. Let me know if you think it is not and I'll remove it.